src/Abp/WebHooks/WebHookSubscription.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using Abp.Application.Services.Dto;
+
+namespace Abp.WebHooks
+{
+    public class WebHookSubscription : EntityDto<Guid>
+    {
+        /// <summary>
+        /// Tenant id of the subscribed.
+        /// </summary>
+        public int? TenantId { get; set; }
+        
+        /// <summary>
+        /// Subscription webhook endpoint
+        /// </summary>
+        public string WebHookUri { get; set; }
+
+        /// <summary>
+        /// Webhook secret
+        /// </summary>
+        public string Secret { get; set; }
+
+        /// <summary>
+        /// Is subscription active
+        /// </summary>
+        public bool IsActive { get; set; }
+
+        /// <summary>
+        /// Subscribed webhook definitions unique names.
+        /// </summary>
+        public List<string> WebHookDefinitions { get; set; }
+
+        /// <summary>
+        /// Gets a set of additional HTTP headers.That headers will be sent with the webhook.
+        /// </summary>
+        public IDictionary<string, string> Headers { get; set; }
```

Comment:
we probably don't need to store headers in the database, it should be configured at runtime (e.g. in sender's implementation)

Reply:
It stores additional headers which tenant adds by subscription. They can receive webhook with different headers by endpoint. It depends on subscription.